include/curl/multi.h

```diff
@@ -448,6 +451,9 @@ typedef int (*curl_push_callback)(CURL *parent,
                                   struct curl_pushheaders *headers,
                                   void *userp);
 
+/* value for MAXIMUM CONCURRENT STREAMS upper limit */
+#define INITIAL_MAX_CONCURRENT_STREAMS ((1U << 31) - 1)
```

Comment:
why not int-max

Reply:
Yeah, I kept this inline to define done at nghttp2, so that it remains line with exact number considering 32 bit arch.